client/lib/react-player/README.md

```diff
@@ -0,0 +1,143 @@
+ReactPlayer
```

Comment:
We should update this README.md with what our app is going to do. React Player is a major component of our app but we can also mention about the broadcasting feature we have.